contracts/Conference.sol

```diff
@@ -252,6 +226,20 @@ contract Conference is GroupAdmin {
         attendanceMaps = _maps;
         ended = true;
         endedAt = now;
+
+        // calculate total attended
+        totalAttended = 0;
+        for (uint i = 0; i < attendanceMaps.length; i++) {
```

Comment:
I remember that Ricardo suggested to add limit of max array size. Should we add something like 1000 for now?

Reply:
100 would do, that equals 25,600 people!